CHANGELOG.md

```diff
@@ -20,4 +20,5 @@
 - Added documentation for [example/standalone](example/standalone) in README.md #8
 
 ### Changed
-- Update vagrant-box, `version = ">= 0.7, < 0.8"` #16
\ No newline at end of file
+- Update vagrant-box, `version = ">= 0.7, < 0.8"` #16
+- Update vagrant-box, `version = ">= 0.9, < 0.10"` #32
```

Comment:
I don't think the description should be here, since `0.1.0` is already been released. I suggest to move the sentence to `0.1.1 UNRELEASED`-section:
```
# Changelog

## [0.1.1 UNRELEASED]
### Added
- Added health checks for nifi service #17
- Added health check for nifi-registry in nifi #28
- Added upstream to nifi registry #26
- Added make proxy-nifi-reg #26

### Changed
- Removed unused files #19
- Update vagrant-box, `version = ">= 0.9, < 0.10"` #32

## [0.1.0]

### Added
- Added 02_run_terraform.yml #6
- Created a simple example for nifi #2
- Added documentation for [example/standalone](example/standalone) in README.md #8

### Changed
- Update vagrant-box, `version = ">= 0.7, < 0.8"` #16
```

Reply:
done